### DISCUSSION TITLE
Проблема с получением 0,25 xch

### DISCUSSION THREAD

**Discussion by stalino1945** [Chia-Network/chia-blockchain#8306] [Support]
Добрый день, сегодня нашел блок на space pool, но почему то вознаграждение 0.25 не поступило на кошелек
номер блока 799298.
Остальные выплаты с самого пула регулярно поступают.

**Comment by loppefaaret** [comment]
https://www.chiaexplorer.com/blockchain/block/0xff8a6f5e5e71e02f255fd15cb74e0fdf0dc864643c9b3e052ae65e898c8e04a6
shows that the farmer reward was `xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p`
that farmer address seems to belong to poolharvest.io
so i guess that is their own farmer running on the pool perhaps ?

**Reply by stalino1945** [reply]
Спасибо за ответ, получается я теперь никак не могу доказать . что это мое вознаграждение? По простому говоря кто-то украл их у меня подменив адрес кошелька?

**Reply by loppefaaret** [reply]
on the farmer, you can check the current address that the farmer portion of the block wins get directed too
Either in the chia app, under the "Farm" tab, in top right corner, under the 3 `...` the "Farmer rewards targer address"
or from `config.yaml` under the `farmer:` section, the address noted in the `xch_target_address:`
this address will be the target of the 0.25 reward from your farmer!
you can then take that address, and look it up on either xchscan.com or chiaexplorer.com
that would confirm when you won blocks
it is possible that this address is pointing at an address that you do not currently control - if you had other keys in the client first, and later on removed these keys... 
it is also possible, though unlikely, that someone managed to hack your machine, and input their own address

**Reply by stalino1945** [reply]
Спасибо за ответ. я уже все везде заменил. Свой выигрыш я уже не верну? Меня это интересует?

**Reply by loppefaaret** [reply]
unless you have had other keys imported into the farmer machine, and removed them. if you have the 24 words for old keys, you could try and import them, and see if there is any rewards in their wallets

**Comment by Annaberdi86** [comment]
did you solve this problem?

**Reply by loppefaaret** [reply]
its starting to become a daily thing that a space pool user is here claiming that the pool is saying they won a block they havent, i assume you also got told so ?

**Comment by Aleksandr898** [comment]
Здраствуйте подскажите как получить монеты Chia , отправил на кошелек и не получил их

**Comment by cintock** [comment]
Добрый день.
У меня возникла та же самая проблема. Фармил на пуле. Ежедневные выплаты приходили, а когда нашел блок, то 0.25 за блок в статистке space.pool отображалось, а сами монеты не пришли на счет.
Спасибо, тем, кто ответил в этой теме. Был дан полезный совет, которым воспользовался. И решил проблему.

На машине, где фармил, раньше были введены другие ключи, когда еще фармил на плотах без пулов. Это было давно и эти ключи из программы chia давно удалил.  Ввел на другом компе мнемонику от старых плотов. И был очень рад, что в этом кошельке нашел свои 0.25 xch, которые пришли туда за найденный блок.

В программе chia, определенно, есть ошибка, что она после ввода нового ключа (и удаления старого), не поменяла автоматически адреса для получения вознаграждения за блок. Или нужно было уведомление сделать на главном экране, что нет приватного ключа к адресам на которые получаем выплаты (чтобы узнать об этом до того (а не после), как монеты ушли не туда). Хорошо, что у меня была мнемоника от старого адреса, иначе эти монеты были бы потеряны навсегда. Возможно, в новой версии это исправлено, не проверял, на машине для фарминга версия chia 1.2.3.
